### ISSUE TITLE
ReferenceError: window is not defined after update to 1.1.0 version

### ISSUE THREAD

**Issue by andreydro** [arunghosh/react-pin-input#198]
I use Next.js (v. 11.1.2)
After updating `react-pin-input` to the `1.1.0` version yarn build started to fail with the following error.
```
ReferenceError: window is not defined
    at Object.<anonymous> (/Users/dro/work/rubyroid/italbazar/frontend/node_modules/react-pin-input/dist/bundle.js:1:216)
    at Module._compile (node:internal/modules/cjs/loader:1103:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1157:10)
    at Module.load (node:internal/modules/cjs/loader:981:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
    at Module.require (node:internal/modules/cjs/loader:1005:19)
    at require (node:internal/modules/cjs/helpers:102:18)
    at Object.17049 (/Users/dro/work/rubyroid/italbazar/frontend/.next/server/pages/_app.js:3804:18)
    at __webpack_require__ (/Users/dro/work/rubyroid/italbazar/frontend/.next/server/webpack-runtime.js:25:42)
    at Object.57080 (/Users/dro/work/rubyroid/italbazar/frontend/.next/server/pages/_app.js:718:33) {
  type: 'ReferenceError'
}
```

Rollback to version `1.0.1` resolves the issue

**Comment by arunghosh** [comment]
Didn't take SSR into consideration. Sorry for this. Will look into this.

**Comment by pravosleva** [comment]
Will works after downgrade to `@1.0.x`... =(

```bash
yarn add react-pin-input@1.0.x
```

**Comment by arunghosh** [comment]
The issue is fixed and available in `v1.3.0`
